common/build.sh

```diff
@@ -8,7 +8,9 @@ popd > /dev/null
 
 cd "$P"
 go install ./... 2> /dev/null || true
-make clean
+if [ -z "${NOCLEAN}" ]; then
+    make clean
+fi
```

Comment:
For reviewers: I'm tempted to make this the default instead of requiring an env. variable to be set. Is it only useful to peeps who run a different arch. in the VM compared to the host?